lib/charms/lego_base_k8s/v0/lego_client.py

```diff
@@ -316,6 +320,22 @@ def _cmd(self) -> List[str]:
             "run",
         ]
 
+
+
+    @property
+    def _app_environment(self) -> Dict[str, str]:
+        """Extract proxy model environment variables."""
+        env = {}
+
+        if (http_proxy := get_env_var(env_var="HTTP_PROXY")):
```

Comment:
Is this the right env var that we would see? Looking at the github runner app, the env variable is `JUJU_CHARM_HTTP_PROXY`, not `HTTP_PROXY`

Reply:
You are right, fixed